src/main/java/com/ftpix/sherdogparser/parsers/FighterParser.java

```diff
@@ -193,6 +248,13 @@ public Fighter parseDocument(Document doc) throws IOException {
     }
 
 
+    private int defineLossKO(Elements methods) {
```

Comment:
We need to have javadoc to all the methods.

and i think it'll be clearer if this methods returns a boolean and be names something like hasOther

Reply:
actually the code could simply be 
```suggestion
    private boolean hasOtherMethod(Elements methods) {
        return methods.get(3).html().contains("OTHER")
    }
```